### PR TITLE
sensors: support current values

### PR DESCRIPTION
**Pull request by clemensg**
Add support for current sensor data collection and add ifdefs for old lm-sensors versions.

### CODE REVIEW

File: src/sensors.c [hunk 1]
```diff
@@ -367,6 +367,9 @@ static int sensors_load_conf(void) {
       if ((feature->type != SENSORS_FEATURE_IN) &&
           (feature->type != SENSORS_FEATURE_FAN) &&
           (feature->type != SENSORS_FEATURE_TEMP) &&
+#if (SENSORS_API_VERSION >= 0x402)
```
Comment (rubenk) [inline src/sensors.c]: There's no need for the round brackets around the preprocessor expression.

File: src/sensors.c [hunk 2]
```diff
@@ -383,6 +386,9 @@ static int sensors_load_conf(void) {
         if ((subfeature->type != SENSORS_SUBFEATURE_IN_INPUT) &&
             (subfeature->type != SENSORS_SUBFEATURE_FAN_INPUT) &&
             (subfeature->type != SENSORS_SUBFEATURE_TEMP_INPUT) &&
+#if (SENSORS_API_VERSION >= 0x402)
```
Comment (rubenk) [inline src/sensors.c]: Same here.

File: src/sensors.c [hunk 3]
```diff
@@ -513,6 +519,10 @@ static int sensors_read(void) {
       type = "temperature";
     else if (fl->feature->type == SENSORS_FEATURE_POWER)
       type = "power";
+#if (SENSORS_API_VERSION >= 0x402)
```
Comment (rubenk) [inline src/sensors.c]: And here.

**Comment by clemensg** [comment]
Ah right, thanks! I updated the PR.

**Pull request merged**